notebooks/2.0-ijd-eda.ipynb

```diff
@@ -80,7 +80,7 @@
    "outputs": [],
```

Comment:
<p>Don't we need the sep='\t' argument now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>yep, added</p>